uaclient/status.py

```diff
@@ -1,34 +1,23 @@
 class TxtColor:
-    HEADER = '\033[95m'
-    OKBLUE = '\033[94m'
     OKGREEN = '\033[92m'
     DISABLEGREY = '\033[37m'
-    WARNING = '\033[93m'
     FAIL = '\033[91m'
     ENDC = '\033[0m'
-    BOLD = '\033[1m'
-    UNDERLINE = '\033[4m'
 
 
 ACTIVE = 'active'
 INACTIVE = 'inactive'
 INAPPLICABLE = 'n/a'
 ENTITLED = 'entitled'
-EXPIRED = 'expired'
 NONE = 'none'
-STANDARD = 'standard'
-ADVANCED = 'advanced'
 
 # Colorized status output for terminal
 STATUS_COLOR = {
     ACTIVE: TxtColor.OKGREEN + ACTIVE + TxtColor.ENDC,
     INACTIVE: TxtColor.FAIL + INACTIVE + TxtColor.ENDC,
     INAPPLICABLE: TxtColor.DISABLEGREY + INAPPLICABLE + TxtColor.ENDC,
     ENTITLED: TxtColor.OKGREEN + ENTITLED + TxtColor.ENDC,
-    EXPIRED: TxtColor.FAIL + EXPIRED + TxtColor.ENDC,
     NONE: TxtColor.DISABLEGREY + NONE + TxtColor.ENDC,
-    STANDARD: TxtColor.OKGREEN + STANDARD + TxtColor.ENDC,
```

Comment:
Good to know, this is a bug. These should have been used by my supportLevel branch which is landed. 

I've filed #538 to capture that.